data/events/2022-fortaleza.yml

```
@@ -18,11 +18,11 @@ cfp_date_start: 2021-10-14T10:00:00-03:00 # start accepting talk proposals.
 cfp_date_end: 2021-11-15T07:00:00-03:00 # close your call for proposals.
```

```suggestion
cfp_date_end: 2022-08-10T07:00:00-03:00 # close your call for proposals.
```
Please confirm this matches your CFP, and update the start and announce dates :)

---

Hello @glasnt, This date is correct. Thanks for your comment :smile: .